extras/Dockerfile.python-packages

```diff
@@ -9,7 +9,7 @@ RUN apt-get update -yq && \
     python3-venv python3-wheel libffi-dev && \
     python3 -m pip install --upgrade pip && \
     python3 -m pip install --upgrade setuptools && \
-    pip install pyxattr grpcio 2>/dev/null >/dev/null && \
+    pip install xattr pyxattr grpcio 2>/dev/null >/dev/null && \
```

Comment:
I wasn't sure about the bundled `xattr` in `pyxattr` and it failed pylint checks so i had to install xattr which pulls the latest once and will be used by pyxattr, however if we are sure that `ubuntu:20.04` will not have this compatibility issue then this can be removed from here.

```
# pip list | grep xattr

# pip install pyxattr
Collecting pyxattr
  Using cached pyxattr-0.7.2-cp38-cp38-linux_x86_64.whl
Installing collected packages: pyxattr
Successfully installed pyxattr-0.7.2

# pip list | grep xattr
pyxattr           0.7.2

# pylint --disable=W0511 -s n server/glusterfsd.py
************* Module glusterfsd
server/glusterfsd.py:12:0: E0401: Unable to import 'kadalulib' (import-error)
server/glusterfsd.py:34:8: I1101: Module 'xattr' has no 'setxattr' member, but source is unavailable. Consider adding this module to extension-pkg-allow-list if you want to perform analysis based on run-time introspection of living objects. (c-extension-no-member)
server/glusterfsd.py:35:14: I1101: Module 'xattr' has no 'getxattr' member, but source is unavailable. Consider adding this module to extension-pkg-allow-list if you want to perform analysis based on run-time introspection of living objects. (c-extension-no-member)
server/glusterfsd.py:53:8: I1101: Module 'xattr' has no 'setxattr' member, but source is unavailable. Consider adding this module to extension-pkg-allow-list if you want to perform analysis based on run-time introspection of living objects. (c-extension-no-member)
server/glusterfsd.py:54:35: I1101: Module 'xattr' has no 'XATTR_CREATE' member, but source is unavailable. Consider adding this module to extension-pkg-allow-list if you want to perform analysis based on run-time introspection of living objects. (c-extension-no-member)

# pip install xattr
Collecting xattr
  Using cached xattr-0.9.7-cp38-cp38-linux_x86_64.whl
Requirement already satisfied: cffi>=1.0.0 in /root/toolbox/venv/kadalu/lib/python3.8/site-packages (from xattr) (1.14.5)
Requirement already satisfied: pycparser in /root/toolbox/venv/kadalu/lib/python3.8/site-packages (from cffi>=1.0.0->xattr) (2.20)
Installing collected packages: xattr
Successfully installed xattr-0.9.7

# pip list | grep xattr
pyxattr           0.7.2
xattr             0.9.7

# pylint --disable=W0511 -s n server/glusterfsd.py
************* Module glusterfsd
server/glusterfsd.py:12:0: E0401: Unable to import 'kadalulib' (import-error)
```

\* E0401 can be disregarded here

Reply:
Validated by pulling docker image and it had no issues, reverting this code.